src/test/java/com/apperian/api/ApiTesting.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Properties;
 
 public class ApiTesting {
-    private static final String PROPERTIES_FILE_NAME = "com/apperian/api/test-configuration.properties";
+    private static final String PROPERTIES_FILE_NAME = "com/apperian/api/test-configuration.properties.test";
```

Comment:
I think this is not really needed

Reply:
that was a mistake, should not have been added